Kryptonite/Keypair+SSHFormat.swift

```diff
@@ -189,6 +219,15 @@ extension DigestType {
 // MARK: SSH Signature Format
 extension KeyPair {
     func signAppendingSSHWirePubkeyToPayload(data:Data, digestType:DigestType) throws -> String {
+        guard let sshKeyPair = self as? SSHCustomSignableFormat
```

Comment:
Something slightly safer might be switching on `publicKey.keyType`, that way we get a compilation error here when another one is added.

Reply:
Yeah I like that better